doc/source/contributing/translate.md

```diff
@@ -0,0 +1,27 @@
+(translate)=
+# Translate ArviZ website and documentation
+You can contribute to ArviZ by helping with translations of our website and
+documentation.
+
+:::{important}
+Helping out with translations doesn't need a GitHub account,
+everything happens on [Transifex](https://www.transifex.com/arviz/)
+:::
+
+To help with translations you will need to go to [Transifex](https://www.transifex.com/arviz/)
```

Comment:
Thanks for checking! It worked for me because I was logged in :facepalm: 

I wanted the link to be project level (in case we add new projects to translate the python docs, or educational material repo...). I think the right alternative for this would be https://www.transifex.com/arviz/public/. Can you confirm it works?